### PR TITLE
[CWS-695] Remove security profiles documentation

### PR DESCRIPTION
**Pull request by DominicBurkart**
### What does this PR do? What is the motivation?

- The security products are sunsetting public-facing security profiles. This PR removes references to security profiles.
- The security products are sunsetting public-facing runtime anomaly detection. This PR removes references to runtime anomaly detection in the public docs.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/ja/security/cloud_workload_security/backend.md [hunk 1]
```diff
@@ -3199,4 +3199,4 @@ CWS のログは、以下の JSON スキーマを持ちます。
 
 
 [1]: /ja/security/cloud_workload_security/
-[2]: /ja/security/cloud_workload_security/agent_expressions
\ No newline at end of file
+[2]: /ja/security/cloud_workload_security/agent_expressions
```
Comment (diogocp) [inline content/ja/security/cloud_workload_security/backend.md:3202]: weird diff 😕

**Review by diogocp — APPROVED** [review]

**Comment by hestonhoffman** [comment]
Thanks for the PR. I've created a ticket and a docs person will pick this up for review.

File: content/ja/security/cloud_workload_security/backend.md (hunk 1 above)
Comment (DominicBurkart) [reply content/ja/security/cloud_workload_security/backend.md:3202]: yeah the "rich diff" doesn't recognize this line as changed; not sure if this is a github regression or if it's tracking something real.

Comment (diogocp) [reply content/ja/security/cloud_workload_security/backend.md:3202]: I bet you added a Mongolian vowel separator just to mess with me

Comment (DominicBurkart) [reply content/ja/security/cloud_workload_security/backend.md:3202]: That would have shown up on the rich diff. 💚

**Review by drichards-87 — APPROVED** [review]
This looks good. Noting for future reference that we don't need to update translated docs as those changes will happen automatically through our translation pipeline. I'll get this merged now.

**Comment by DominicBurkart** [comment]
👋 I believe that we need to retain the documentation of configuration for security profiles and runtime anomaly detection on the agent (not deleted in this PR), cc @paulcacheux for confirmation.

**Comment by paulcacheux** [comment]
I have honestly no idea, can we ask in a slack channel so that we can bring PMs ?

**Comment by drichards-87** [comment]
@DominicBurkart By retain, do you mean revert the updates to the setup docs? Because this PR removed it from the code samples. Or are you asking that we keep a copy of this available for future reference?

**Comment by DominicBurkart** [comment]
> By retain, do you mean revert the updates to the setup docs? 

No, I just meant that we should keep e.g. this doc, which is not marked for deletion in this PR: https://docs.datadoghq.com/security/threats/backend/#securityprofilecontext

Paul will followup if those docs also need to be deleted!

**Comment by drichards-87** [comment]
Ah, okay, that makes sense. Thanks!

**Pull request merged**